src/components/Tab/Tab.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+import { classNames } from '@shopify/react-utilities/styles';
+import { TAB } from '../ThemeIdentifiers';
+import * as baseTheme from './Tab.scss';
+
+// All prototypes type
+export interface Props {
+  // Description of tab (like title only, title with icon, or any other react component)
+  tabDescription?: React.ReactNode;
+  // Unique identity for tab
+  tabId: string;
+  // Identity of selected/currently active tab
+  activeTabId?: string;
+  // Set theme for tab
+  theme?: any;
+  children?: any;
```

Comment:
@HiirenP No need to define children prop as its a pre-defined prop which comes with every component, so we don't have to define its type

Reply:
resolved!